node/defaults.go

```diff
@@ -28,12 +28,15 @@ import (
 )
 
 const (
-	DefaultHTTPHost    = "localhost" // Default host interface for the HTTP RPC server
-	DefaultHTTPPort    = 8545        // Default TCP port for the HTTP RPC server
-	DefaultWSHost      = "localhost" // Default host interface for the websocket RPC server
-	DefaultWSPort      = 8546        // Default TCP port for the websocket RPC server
-	DefaultGraphQLHost = "localhost" // Default host interface for the GraphQL server
-	DefaultGraphQLPort = 8547        // Default TCP port for the GraphQL server
+	DefaultHTTPHost         = "localhost" // Default host interface for the HTTP RPC server
+	DefaultHTTPPort         = 8545        // Default TCP port for the HTTP RPC server
+	DefaultWSHost           = "localhost" // Default host interface for the websocket RPC server
+	DefaultWSPort           = 8546        // Default TCP port for the websocket RPC server
+	DefaultGraphQLHost      = "localhost" // Default host interface for the GraphQL server
+	DefaultGraphQLPort      = 8547        // Default TCP port for the GraphQL server
+	DefaultHTTPTimeoutRead  = 30          // Default HTTP Read request timeout in seconds
```

Comment:
Another possibility is to change it here, but that will generate a variable that references another variable which makes de code a little harder to read.
But as is it, if we change the defaults, we will have to remember to also change these values, that's why I prefer the other way